default_metadata/definition/checkbox.json

```diff
@@ -0,0 +1,7 @@
+{
+	"_id": "definition.checkbox",
+	"_type": "checkbox",
+	"label": "Option",
+	"hint": "[Hint text (optional)]",
+	"value": "checkbox-value"
+}
```

Comment:
@brenetic can you confirm if this definition file should be on the schema?

Reply:
Oh wait, I think this file is not needed.

Reply:
for collections we would have:

- schema for the collection
- schema for the collection item
- default metadata for the collection
- default metadata for the collection item

Reply:
Oh I see. True. Then we can access the attribute of a new item via `DefaultMetadata['definition.checkbox']`
I think the word definition just confused me. Apologies.

Reply:
Thanks for the clarification

Reply:
we do need the default metadata for a collection item as we will be inserting it into a button